### PR TITLE
Lt1000 docs

### PR DESCRIPTION
**Pull request by pietrushnic**
@artur-rs I'm setting this PR to have easier way of analyzing what was done. I'm not sure if this research would be accepted upstream, but it may make sense for other people, if they want to integrate.

### CODE REVIEW

**Comment by artur-rs** [comment]
@pietrushnic I think that this documentation is rather for the internal use and demo purposes/reproduction

**Comment by pietrushnic** [comment]
@artur-rs yes, but the repository is public. We should probably ask Richard if he needs some this kind of documentation and the decide if it is helpful any way. If not we should commit that to internal repository.

File: docs/fwupd-demo-doc.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+After successful installation there is possibility to run daemon in --verbose
+(debug) mode: `sudo /usr/local/libexec/fwupd/fwupd --verbose` (one terminal will work as
+debug monitor, run commands in another).
+
+LVFS
+====
+
+The metadata URLs can be used in `/usr/local/etc/fwupd/remotes.d/` to perform
+end-to-end tests. It is important to not share the embargo URL with external
+users if you want the firmware to remain hidden from the public. You also may
```
Comment (pietrushnic) [inline docs/fwupd-demo-doc.md]: @artur-rs you just did that above :)

File: docs/fwupd-demo-doc.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+need to do `fwupdmgr refresh` on each client to show new updates.
+
+|Description    |Public|URL                    |Custom Remote     |
+|:-------------:|:----:|:---------------------:|:----------------:|
+|Stable         |	Yes  |firmware.xml.gz        |not required      |
+|Testing        | Yes  |firmware-testing.xml.gz|not required      |
+|Embargo ‘3mdeb’| No   |firmware-3c81bfd       |3mdeb-embargo.conf|
+
+> End-to-end tests using the metadata can only be used with a custom /usr/local/etc/fwupd/remotes.d/3mdeb-embargo.conf file as this user account
```
Comment (pietrushnic) [inline docs/fwupd-demo-doc.md]: @artur-rs please wrap

**Pull request closed without merging**